moped-editor/src/utils/mapHelpers.js

```diff
@@ -616,11 +616,11 @@ export function useFeatureCollectionToFitBounds(
      */
     const fitViewportToBounds = viewport => {
       if (featureCollection.features.length === 0) return viewport;
-
+      debugger;
```

Comment:
@sergiogcx, do we need to remove this `debugger` from the release canddiate?

Reply:
Ah, yes my bad. I forgot to push my latest commit.

Reply:
@mateoclarke done